backend/execution_cache/src/mutation.go

```diff
@@ -0,0 +1,70 @@
+package main
+
+import (
+	"crypto/sha256"
+	"encoding/hex"
+	"fmt"
+	"log"
+
+	"k8s.io/api/admission/v1beta1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	ArgoWorkflowTemplate string = "workflows.argoproj.io/template"
+	ExecutionKey         string = "pipelines.kubeflow.org/execution_cache_key"
+	AnnotationPath       string = "/metadata/annotations"
+)
+
+const (
+	Add OperationType = "add"
```

Comment:
Minor, but I think this should be named `OperationTypeAdd`.

It's common practice, when defining such types to use the type (or something that reminds of the type) as prefix. E.g., see https://github.com/argoproj/argo/blob/79b09ed43550bbf958c631386f8514b2d474062c/pkg/apis/workflow/v1alpha1/workflow_types.go#L16-L66

Also, this definition should live next to the `OperationType` type definition (I.e., `admission.go` file)